### ISSUE TITLE
WCAG contrast fail on Markdown quote box

### ISSUE THREAD

**Issue by HonkingGoose** [prettier/prettier#11408]
## What's the problem?

Text contrast for "quote" box/admonition is too low, it has a contrast ratio of `4.46` which fails the WCAG contrast check.
For general usability you should aim for the AA rating at least.

## Steps to reproduce:

1. Go to published website on this page: https://prettier.io/docs/en/options.html#print-width
1. Open developer tools in the browser
1. Use the accessibility tool and hover over the text to see the contrast ratio `4.46`, fails WCAG contrast check.

## Links to relevant source code files:

Here is the source for the published markdown: https://github.com/prettier/prettier/blob/main/website/versioned_docs/version-stable/options.md#print-width

I'm not sure where the colors are coming from, they could be coming from the default Docusaurus 1 theme, or from some plugin/extension/custom CSS override.

## Proposed solution:

I recommend you follow Steve Schoger's great advice on dealing with text on a colored background:

> Achieving an accessible contrast ratio is very difficult when using white text on a colored background.
>
> Using dark colored text on a soft colored background is much easier to make accessible, and usually looks better to boot

![image](https://user-images.githubusercontent.com/34918129/131213764-39c2ef87-f276-48ca-8e4f-4ff3c50ced80.png)

Source: https://twitter.com/steveschoger/status/1045371124274483200/

So a lighter yellow for the background, and a dark yellow for the text itself will probably pass the WCAG color checks, and look way nicer as well.
